app/services/schedule_service.rb

```diff
@@ -8,11 +8,27 @@ def initialize(edition)
   end
 
   def schedule(user: nil, reviewed: false)
-    scheduling = Scheduling.new(pre_scheduled_status: edition.status, reviewed: reviewed)
+    scheduling = Scheduling.new(pre_scheduled_status: edition.status,
+                                reviewed: reviewed)
+
     set_edition_status(scheduling, user)
     create_timeline_entry(scheduling)
 
-    create_publish_intent(edition)
+    create_publish_intent
+    schedule_to_publish
+  end
+
+  def reschedule(user: nil)
+    unless edition.scheduled?
+      raise "Edition must be scheduled in order to reschedule"
+    end
+
+    scheduling = edition.status.details
+    set_edition_status(scheduling, user)
+    create_timeline_entry(scheduling)
+
+    create_publish_intent
```

Comment:
Are we using create to replace since the eventual request is a put? If so I wonder if we'd want to rename the method.

Reply:
`create_or_update...` looked weird to me, so I've gone with an alias. Would you prefer just a method name change?